test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -195,5 +255,28 @@ private static HttpContext GetHttpContext()
 
             return httpContext;
         }
+
+        private class ContentController : Controller
+        {
+            public IActionResult Content_WithNoEncoding()
+            {
+                return Content("Hello!!");
+            }
+
+            public IActionResult Content_WithEncodingInCharset()
+            {
+                return Content("Hello!!", "text/xml; charset=us-ascii; p1=p1-value");
+            }
+
+            public IActionResult Content_WithInvalidCharset()
+            {
+                return Content("Hello!!", "text/xml; charset=invalid; p1=p1-value");
+            }
+
+            public IActionResult Content_WithEncodingInCharset_AndEncodingParameter()
+            {
+                return Content("Hello!!", "text/xml; charset=invalid; p1=p1-value", Encoding.ASCII);
+            }
```

Comment:
These are tests for the controller methods, not for `ContentResult`. These should go somewhere else.